content/posts/using-the-redis-allocator-in-rust.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Using the Redis Allocator in Rust"
+date: 2019-11-12
+authors:
+  - Gavrie Philipson
+tags: ["redisjson", "rust"]
+---
+
+## Introduction
+
+While developing [redis-module-rs](https://github.com/RedisLabsModules/redismodule-rs), the [Rust](https://www.rust-lang.org) API for writing [Redis modules](https://redis.io/topics/modules-intro), I encountered the need to set up a custom memory allocator.
```

Comment:
```suggestion
While developing [redismodule-rs](https://github.com/RedisLabsModules/redismodule-rs), the [Rust](https://www.rust-lang.org) API for writing [Redis modules](https://redis.io/topics/modules-intro), I encountered the need to set up a custom memory allocator.
```

Reply:
We actually renamed it when we published it to crates.io, since the redismodule name was already taken and the author did not cooperate.